src/test/java/uk/gov/pay/directdebit/payments/services/GoCardlessServiceTest.java

```diff
@@ -49,6 +50,7 @@
     static final String TRANSACTION_ID = "sdkfhsd2jfhjdks";
     static final SortCode SORT_CODE = SortCode.of("123456");
     static final AccountNumber ACCOUNT_NUMBER = AccountNumber.of("12345678");
+    static final GoCardlessCreditorId CREDITOR_ID = GoCardlessCreditorId.of("gocardless-test-creditor-id-here");
```

Comment:
This doesn’t actually ever seem to be used

Reply:
Nice spot! We forgot to remove it after renaming.